### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by zaidansari42**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+}
+
+section {
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  border: 5px solid;
+  min-height: 100vh;
+  background: #112d42;
+  display: flex;
```
Comment (mujeeb100) [inline styles.css]: make it group similar properties

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,319 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+* {
```
Comment (syedabdullahali) [inline styles.css:3]:
```
 margin: 0;
  padding: 0;

  box-sizing: border-box;

  font-family: "Poppins", sans-serif;
```

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+section {
+  position: relative;
```
Comment (syedabdullahali) [inline styles.css:13]:
```
 position: relative;

  display: flex;
  justify-content: center;
  align-items: center;

  background: #112d42;
  border: 5px solid;
  min-height: 100vh;
```

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+  height: 45px;
+}
+
+section .container .contactinfo .sci li {
+  list-style: none;
+  margin: 0 15px;
+}
+
+section .container .contactinfo .sci li a img {
+  /* max-width: 50px; */
```
Comment (murtazshaikh) [inline styles.css]: remove this comment if not required

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+  position: relative;
+  cursor: pointer;
+  background: #0f3f59;
+  color: #fff;
+  border: none;
+  max-width: 150px;
+  padding: 12px;
+}
+
+section .container .contact-form .formbox .inputbox input[type="submit"]:hover {
```
Comment (murtazshaikh) [inline styles.css]: In CSS selectors no need to select it from its parent, just select last 2 or 3 selectors so it can be specific than other selectors.

**Review by murtazshaikh — APPROVED** [review]

File: styles.css (hunk 4 above)
Comment (zaidansari42) [reply styles.css]: Yes sure

File: styles.css (hunk 5 above)
Comment (zaidansari42) [reply styles.css]: Before, I used to select by 2-3 selectors only, But in the Youtube Video he used this way, so I thought maybe this is the correct way to select it to make sure u dont select anything else (clashing).

**Review by syedabdullahali — APPROVED** [review]

File: styles.css (hunk 1 above)
Comment (zaidansari42) [reply styles.css]: Done

File: styles.css (hunk 3 above)
Comment (zaidansari42) [reply styles.css:13]: Done

File: styles.css (hunk 2 above)
Comment (zaidansari42) [reply styles.css:3]: Done

**Pull request merged**